packages/editor/src/components/post-card-panel/index.js

```diff
@@ -30,6 +31,24 @@ import {
 } from '../../store/constants';
 import { unlock } from '../../lock-unlock';
 import TemplateAreas from '../template-areas';
+import { viewPostAction } from '../post-actions/actions';
+
+function ActionTrigger( { action, item } ) {
+	if ( ! action.isEligible( item ) ) {
+		return null;
+	}
+	return (
+		<Button
+			label={ action.label }
+			icon={ action.icon }
+			isDestructive={ action.isDestructive }
+			size="compact"
```

Comment:
You'll need to update the size to `small` to match the designs. With this change you don't need the css changes you made [here](packages/editor/src/components/post-card-panel/style.scss), so you can remove them.